tests/test-rest-nav-menu-items-controller.php

```diff
@@ -23,85 +45,760 @@ public static function wpSetUpBeforeClass( $factory ) {
 		);
 	}
 
-	function setUp() {
+	/**
+	 *
+	 */
+	public static function wpTearDownAfterClass() {
+		self::delete_user( self::$admin_id );
+		self::delete_user( self::$subscriber_id );
+	}
+
+	/**
+	 *
+	 */
+	public function setUp() {
 		parent::setUp();
+
+		$this->tag_id = self::factory()->tag->create();
+
 		$this->menu_id = wp_create_nav_menu( rand_str() );
+
+		$this->menu_item_id = wp_update_nav_menu_item(
+			$this->menu_id,
+			0,
+			array(
+				'menu-item-type'      => 'taxonomy',
+				'menu-item-object'    => 'post_tag',
+				'menu-item-object-id' => $this->tag_id,
+				'menu-item-status'    => 'publish',
+			)
+		);
 	}
 
+	/**
+	 *
+	 */
 	public function test_register_routes() {
+		$routes = rest_get_server()->get_routes();
+
+		$this->assertArrayHasKey( '/wp/v2/menu-items', $routes );
+		$this->assertCount( 2, $routes['/wp/v2/menu-items'] );
+		$this->assertArrayHasKey( '/wp/v2/menu-items/(?P<id>[\d]+)', $routes );
+		$this->assertCount( 3, $routes['/wp/v2/menu-items/(?P<id>[\d]+)'] );
 	}
 
+	/**
+	 *
```

Comment:
Doesn't really matter, but if we do a `@ticket ???` it might be easier to find+replace when we merge.

Reply:
I think should add that when we move this to core.